content/marketing/content/editorial/editorial-strategy.md

```diff
@@ -99,7 +90,7 @@ The in-depth value of the post (it’s 39 minutes long) likely also makes the pi
 
 Examples are a great opportunity for humor. In the post [What is an A/B Test?](https://netflixtechblog.com/what-is-an-a-b-test-b08cc1b57962), the Netflix Engineering blog authors write “Let’s say — this is a hypothetical! — we want to learn if a new product experience that flips all of the boxart upside down in the TV UI is good for our members” and includes an image of all the boxart on the Netflix home screen upside down.
 
-## Assume devs are knowledgeable
+##### Assume readers are knowledgeable
```

Comment:
Since we might also speak to engineering managers/senior leaders on the blog I removed some of the references to devs specifically